integration-tests/oauth2/src/test/groovy/com/okta/spring/tests/oauth2/implicit/ImplicitLocalValidationGroupIT.groovy

```diff
@@ -37,4 +39,18 @@ class ImplicitLocalValidationGroupIT extends ApplicationTestRunner {
         .then()
             .body(Matchers.equalTo("Everyone has Access: joe.coder@example.com"))
     }
+
+    @Test
+    void test401ResponseBody() {
+         given()
+            .contentType(ContentType.ANY)
+            .redirects()
+                .follow(false)
+        .when()
+            .get("http://localhost:${applicationPort}/everyone")
+        .then()
+            .statusCode(401)
+            .header("WWW-Authenticate", startsWith("Bearer"))
+            .body(Matchers.equalTo("401 UNAUTHORIZED\n"))
```

Comment:
Why does this one need a newline and the previous test didn't? Maybe a `.contains` with a constant?